osler/inventory/forms.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
+from django.forms import ModelForm, Form
+from . import models
+
+from django.forms import (Form, CharField, ModelForm, EmailField,
+                          CheckboxSelectMultiple, ModelMultipleChoiceField, IntegerField)
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Submit
+from crispy_forms.bootstrap import InlineCheckboxes
+from crispy_forms.layout import ButtonHolder, Submit
+
+from crispy_forms.layout import Field
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Layout, Submit, Row, Column
+
+
+class DrugForm(ModelForm):
+    class Meta:
+        model = models.Drug
+        fields = ['name', 'category', 'stock', 'manufacturer', 'lot_number', 'unit', 'dose', 'expiration_date']
+
+    def __init__(self, *args, **kwargs):
+        super(DrugForm, self).__init__(*args, **kwargs)
+
+        self.helper = FormHelper(self)
+        self.helper.form_method = 'post'
+        self.helper.form_class = 'form-horizontal'
+        self.helper.label_class = 'col-lg-2'
+        self.helper.field_class = 'col-lg-8'
+        self.helper.add_input(Submit('submit', 'Submit'))
+
+class DuplicateDrugForm(ModelForm):
+    class Meta:
+        model = models.Drug
```

Comment:
I feel like this doesn't have to be a ModelForm, I don't have a basis for this but it weirds me out that both DuplicateDrugForm and DrugForm are ModelForms for Drug. When we do DuplicatePatientForm elsewhere, it's just a normal Form

Reply:
You can use either Form or ModelForm, but ModelForm lets me use `class Meta:` which lets me just put in a list what fields I want in the form. It just makes it easier imo. Here's more on that: https://stackoverflow.com/questions/2303268/djangos-forms-form-vs-forms-modelform.